### DISCUSSION TITLE
Localization promlem

### DISCUSSION THREAD

**Discussion by NiTiSon** [Pixeval/Pixeval#452] [Ideas]
Is there any chance to _at least_ English localization?

Chinese symbols are nonsense for most peoples in the world

**Comment by Poker-sang** [comment]
Pixeval's localization supports Chinese, Russian and English. You can find the settings here.

![image](https://github.com/Pixeval/Pixeval/assets/62325494/96b076bb-b6a2-4ee1-8108-fdca395379ba)

The vast majority of Pixeval users are Chinese, so we decided to use Chinese as the main language. For most of us, English is also nonsense.

**Comment by NiTiSon** [comment]
There is no option button on login screen

**Comment by Poker-sang** [comment]
I can add it in the next version. BTW the software automatically selects the language based on the system, if you have an English system. Then English will be selected automatically.
